packages/component_code_gen/generate_action.py

```diff
@@ -0,0 +1,27 @@
+import os
+import argparse
+import templates.generate_actions
+import templates.transform_to_action
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--app', '-a', help='the app_name_slug', required=True)
```

Comment:
As I've been playing around with this more, I definitely +1 my comment earlier re: wanting to add a bunch of notes to a Markdown file and then pass that as `--instructions file.md` or `--docs file.md` or something. So I'll put the requirements, relevant API docs, etc. there.